setup.py

```diff
@@ -16,21 +16,23 @@
 
 # This reads the __version__ variable from openfermion/_version.py
 __version__ = ''
-exec(open('src/openfermion/_version.py').read())
+with open('src/openfermion/_version.py') as v:
+    exec(v.read())
 
-# Readme file as long_description:
+# The readme file is used as the long_description:
 long_description = '===========\n' + 'OpenFermion\n' + '===========\n\n'
 with open('README.rst', 'r', encoding='utf-8') as readme:
     long_description += readme.read()
 
-# Read in package requirements.txt
-requirements = open('dev_tools/requirements/deps/runtime.txt').readlines()
+# Read in package requirements.txt.
+with open('dev_tools/requirements/deps/runtime.txt') as r:
```

Comment:
I just ran into this recently when mypy complained about with open syntax, apparently pathlib.Path(file).read_text() is preferred?

Reply:
Hmm. Mypy doesn't complain about this for some reason. I'm inclined to leave it as-is for now, just to get on with this release.

Reply:
Maybe it was a linter, anyway sounds fine.